src/Spice86/ViewModels/BreakpointsViewModel.cs

```diff
@@ -15,6 +15,11 @@ public partial class BreakpointsViewModel : ViewModelBase {
     public BreakpointsViewModel(EmulatorBreakpointsManager emulatorBreakpointsManager) {
         _emulatorBreakpointsManager = emulatorBreakpointsManager;
     }
+
+    public event Action? BreakpointDeleted;
+    public event Action? BreakpointCreated;
```

Comment:
Is this ever Invoked?

Reply:
Created will be invoked once the view has a form to create any type of breakpoint.

Deleted should be.

Reply:
They are listened to in order to update the DISASM view:

        breakpointsViewModel.BreakpointCreated += UpdateDisassemblyInternal;
        breakpointsViewModel.BreakpointDeleted += UpdateDisassemblyInternal;
        breakpointsViewModel.BreakpointDisabled += UpdateDisassemblyInternal;
        breakpointsViewModel.BreakpointEnabled += UpdateDisassemblyInternal;

Reply:
I removed 'created' since it's never invoked.